config/software/psycopg2.rb

```diff
@@ -11,5 +11,11 @@
 
 build do
   ship_license "https://raw.githubusercontent.com/psycopg/psycopg2/master/LICENSE"
-  command "#{install_dir}/embedded/bin/pip install #{name}==#{version}", :env => env
+  # if ohai['platform'] == 'windows'
+  #   command "CMD /C \"SET Path=\\\"#{ENV['PATH']}:\\embedded\\Scripts\\\" && "
+  #           "#{install_dir}/embedded/Scripts/pip install -I #{name}==#{version} "\
+  # else
+  #   command "#{install_dir}/embedded/bin/pip install -I #{name}==#{version}", :env => env
+  # end
```

Comment:
hmm what is this commented out block for? If it's useful we should explain why, otherwise let's delete it